README.md

```diff
@@ -174,10 +174,36 @@ fields | `object[]` | `required` | None | information about room
 
 ### HotelOwner.cs
 
+#### UpdateRoom(fields)
+
+Name    | Type     | Argument     | Default | Description
+--------|----------|--------------|---------|------------
+fields | `object[]` | `required` | None | information for updating room
+
```

Comment:
In [HotelOwner.cs](https://github.com/mezidia/hotel-booking/blob/master/Database/Classes/HotelOwner.cs) there is function _ChangeAccess_, not an _UpdateRoom()_